src/server/tran_server.cpp

```diff
@@ -420,11 +434,7 @@ tran_server::reset_main_connection ()
     }
 
   auto ulock = std::unique_lock<std::shared_mutex> { m_main_conn_mtx };
```

Comment:
Nit.
Wrap this mutex in a scope.

Reply:
Although this lock is kept at the end of this function?

Reply:
The code is correct as it is.
With the extra scope it would just be more explicit.
```
{
  // unique_lock
  // if (main ) { .. }  else { .. }
}

return NO_ERROR.
```

Reply:
OK, I get it.